app/Filament/Resources/LessonResource.php

```diff
@@ -57,7 +58,14 @@ public static function table(Table $table): Table
                 //
             ])
             ->actions([
-                //
+                Tables\Actions\EditAction::make()
+                    ->url(
+                        fn (Pages\ListLessons $livewire, Model $record): string => static::$parentResource::getUrl('lessons.edit', [
+                            'record' => $record,
+                            'parent' => $livewire->parent,
+                        ])
+                    ),
+                Tables\Actions\DeleteAction::make(),
```

Comment:
The table has access to the `$parent` property through `$livewire` (which is the current page list, create, edit etc..)

Reply:
Good catch. This simplifies the actions!